kernel/clojure/src/main/resources/logback.xml

```diff
@@ -0,0 +1,27 @@
+<?xml version="1.0" encoding="UTF-8" ?>
+<configuration debug="false">
+
+  <appender name="STDOUT" class="ch.qos.logback.core.ConsoleAppender">
+    <encoder>
+      <pattern>%date %-5level [%class{0}.%M:%line] %msg%n</pattern>
+    </encoder>
+  </appender>
+
+  <appender name="FILE_CLOJURE" class="ch.qos.logback.core.FileAppender">
+    <encoder>
+      <pattern>%date %-5level [%class{0}.%M:%line] %msg%n</pattern>
+    </encoder>
+    <append>true</append>
+    <file>${user.home}/.jupyter/logs/jupyter-clojure-kernel.log</file>
```

Comment:
don't write to this file, just write to stderr, like the python log from "jupyter notebook".